docs/user/metacat/source/install.rst

```diff
@@ -9,19 +7,25 @@ Instructions for the Linux system is included in this section.
 
 System Requirements
 -------------------
+
 In addition to meeting the recommended system requirements, the server on which
-you wish to install Metacat must have the following software installed and
-running correctly:
+you wish to install Metacat must have the following software installed and running correctly:
```

Comment:
Hmm, I would like to reword it as such instead of specifying a version or adding the sentence to see each section.

> In addition to meeting the recommended system requirements, the server on which you wish to install **the latest version of Metacat** must have the following software installed and running correctly

I have adjusted this and please let me know if you think this is not clear enough (and if so, I think adding the specific version number is better).